productivity-suite/app/legacy-app/src/worker/script.ts

```diff
@@ -173,4 +173,60 @@ gateway.registerFragment({
   },
 });
 
+gateway.registerFragment({
+  fragmentId: "news",
+  // Note: deployment part of the url is fine also for local development since then
+  //       only the path part of the url is being used
+  getBaseUrl: () =>
+    `https://productivity-suite-news-fragment.devdash.workers.dev`,
+  prePiercingStyles: `
+    :not(piercing-fragment-outlet) > piercing-fragment-host[fragment-id="news"] {
+      position: absolute;
+      margin: 0 1rem;
+      top: 13.9rem;
+      left: 0;
+      right: 0;
+    }
+
+    @media (max-width: 45rem) {
```

Comment:
awesome, thanks for the positioning, I hope it wasn't too tedious 🙂 (I think this is quite one dx thing that it's not really the best here 😓)

Reply:
No prob! This was probably the most fun I had in this PR tbh :D